src/Security/Handler/AclSecurityHandler.php

```diff
@@ -164,6 +164,7 @@ public function deleteObjectSecurity(AdminInterface $admin, $object)
     public function getObjectAcl(ObjectIdentityInterface $objectIdentity)
     {
         try {
+            /** @var MutableAclInterface $acl */
```

Comment:
We could open a PR on symfony to improve the returntype of MutableAclProviderInterface::findAcl

Reply:
done https://github.com/symfony/security-acl/pull/60